sw/airborne/modules/relativelocalizationfilter/arrayfunctions.c

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (C) Mario Coppola
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/relativelocalizationfilter/arrayfunctions.c"
+ * @author Mario Coppola
+ * Array functions for array processing
+ */
+
+#include "arrayfunctions.h"
+
+/*
+Shifts an array to the left by a defined number of steps at a time AND copies the left most element back to the right side!
+*/
+void array_shiftleft(float *array, int size, int shift)
+{
+	int i;
```

Comment:
Indentation is wrong again. Please run fix_code_style on all your c and h files.